docs/agents.md

```diff
@@ -166,23 +166,23 @@ print(result.data)
 
 _(This example is complete, it can be run "as is")_
 
-## Retrievers
+## Tools
 
-Retrievers provide a mechanism for models to request extra information to help them generate a response.
+Tools provide a mechanism for models to request extra information to help them generate a response.
 
 They're useful when it is impractical or impossible to put all the context an agent might need into the system prompt, or when you want to make agents' behavior more deterministic or reliable by deferring some of the logic required to generate a response to another (not necessarily AI-powered) tool.
 
-!!! info "Retrievers vs. RAG"
-    Retrievers are basically the "R" of RAG (Retrieval-Augmented Generation) — they augment what the model can do by letting it request extra information.
+!!! info "Tools vs. RAG"
+    Tools are basically the "R" of RAG (Retrieval-Augmented Generation) — they augment what the model can do by letting it request extra information.
```

Comment:
This needs to be updated lol. Maybe we can just drop this callout since we renamed things.